grails-app/services/org/bbop/apollo/PermissionService.groovy

```diff
@@ -499,13 +499,38 @@ class PermissionService {
         return jsonObject
     }
 
+    Boolean hasGlobalPermissions(JSONObject jsonObject, PermissionEnum permissionEnum) {
```

Comment:
I am a little confused about this function. If permissionEnum = ADMINISTRATIVE then it actually checks if the user is global ADMIN. Why we just use `Boolean hasGlobalPermissions(JSONObject jsonObject, PermissionEnum GlobalPermissionEnum)`